src/ray/protobuf/usage.proto

```diff
@@ -89,4 +89,6 @@ enum TagKey {
   // This is to track usage of the buggy behavior that will be fixed.
   // See https://github.com/ray-project/ray/issues/28465 for more details.
   RAY_GET_TIMEOUT_ZERO = 304;
+  // A map from subnet to number of nodes in the subnet.
```

Comment:
Can you write -> "e.g., 127.0.0.1 -> 127.0.0.* will be reported". I think it'd be better to be more concrete since it is external facing